test/e2e_node/summary_test.go

```diff
@@ -116,8 +116,10 @@ var _ = SIGDescribe("Summary API [NodeConformance]", func() {
 				})
 			}
 			expectedMajorPageFaultsUpperBound := 10
+			expectedPageFaults := 1000000
 			if IsCgroup2UnifiedMode() {
 				expectedMajorPageFaultsUpperBound = 1000
+				expectedPageFaults = 1e9
```

Comment:
`1e9` might be a bit big as per https://github.com/kubernetes/kubernetes/issues/105301#issuecomment-928466963 but it's hard to know for sure, and I'm fine with this being bigger side since it's hard to know what exactly to expect.

Reply:
Please add a comment in code on why this value is set here.

Reply:
I am using the same value that is used for the container cgroup at line 109.

Since the cgroupsv2 stats are recursive, I think the value should at least be like for the container.